cpp/src/arrow/engine/substrait/extension_set.cc

```diff
@@ -944,6 +944,26 @@ ExtensionIdRegistry::SubstraitAggregateToArrow DecodeBasicAggregate(
                                call.id().name, " to have at least one argument");
       }
       case 1: {
+        std::shared_ptr<compute::FunctionOptions> options = nullptr;
+        if (arrow_function_name == "stddev" || arrow_function_name == "variance") {
+          auto maybe_dist = call.GetOption("distribution");
```

Comment:
Can you add a link to the substrait spec here:
https://github.com/substrait-io/substrait/blob/73228b4112d79eb1011af0ebb41753ce23ca180c/extensions/functions_arithmetic.yaml#L1240